features/executor-queue.feature

```diff
@@ -0,0 +1,68 @@
+@executorqueue
+Feature: Executor Queue
+
+    Users want their build to enqueue for later execution if all the execution engines are
+    offline or full, so that they do not have to worry about forgotten builds.
+
+    Screwdriver should be pushing planned builds into a queue. And all available executors
+    should be listening on that queue and taking items off when they have capacity.
+
+    Rules:
+        - Executors can determine if they are capable of running a new build.
+        - Multiple instances of the same executor can connect to the queue.
+        - Multiple executor types can connect to the same queue.
+        - JWT is not created until build is sent to the executor.
+        - Users know how far in the queue a build is.
+    
+    @ignore
+    Scenario Outline: Push a build into the queue
+        Given an executor type is <type>
+        When a new build is created
+        Then that build is pushed into the queue
+        
+        Examples:
+            | type   |
+            | k8s    |
+            | docker |
+            | j5s    |
```

Comment:
On a side note (not a blocker here) I really want to rename `j5s` into `jenkins`